src/meshpy/four_c/locsys_condition.py

```diff
@@ -34,25 +37,52 @@ class LocSysCondition(BoundaryCondition):
 
     def __init__(
         self,
-        geometry_set,
-        rotation,
+        geometry_set: GeometrySet,
+        rotation: Rotation,
+        *,
+        function_array=None,
+        update_node_position: bool = False,
+        use_consistent_node_normal: bool = False,
         **kwargs,
     ):
         """Initialize the object.
 
         Args
         ----
```

Comment:
Thanks for updating the doc style (also in other parts of this PR)!

I think this is slightly off the [google doc style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)

```suggestion
        Args:
            geometry_set: ...
            rotation: ...
```